src/pages/settings/Payments/PaymentsPage/BasePaymentsPage.js

```diff
@@ -76,12 +80,25 @@ class BasePaymentsPage extends React.Component {
             this.setMenuPosition();
         }
 
+        // If the user was previously offline, skip debouncing showing the loader
```

Comment:
Do we need to skip debouncing? If we come back online we should fetch data again and show a loader if it takes a while.

Reply:
The issue is that we also grey out the wallet amount when offline, so it does a strange thing where, the number will go from grey to white, then the loader will take over

Reply:
Ok that makes sense. I'm a bit confused still because if you are on the page and go online->offline->online (slow 3g) then you don't see the loader right away. I would say this is NAB at this point but would be nice to fix.


https://user-images.githubusercontent.com/26260477/213533354-05ba9bb5-ded1-4ed4-81d8-c5a1816d1fbd.mov